cli/src/commands/grpc-service/commands/generate.ts

```diff
@@ -104,9 +211,45 @@ type GenerationOptions = {
   spinner: Ora;
   packageName?: string;
   goPackage?: string;
+  javaPackage?: string;
+  javaOuterClassname?: string;
+  javaMultipleFiles?: boolean;
+  csharpNamespace?: string;
+  rubyPackage?: string;
+  phpNamespace?: string;
+  phpMetadataNamespace?: string;
+  objcClassPrefix?: string;
+  swiftPrefix?: string;
   lockFile?: string;
+  operationsDir?: string;
+  queryIdempotency?: string;
+  customScalarMappings?: Record<string, string>;
 };
 
+/**
+ * Read all GraphQL operation files from a directory
+ */
+async function readOperationFiles(operationsDir: string): Promise<string> {
+  const files = await readdir(operationsDir);
+  const operationFiles = files.filter((file) => {
+    const ext = extname(file).toLowerCase();
+    return ext === '.graphql' || ext === '.gql';
```

Comment:
There is also `.graphqls`, `.gqls`

Reply:
OK - I thought that these were for schema files - not operation documents - will update.